builder/virtualbox/common/step_export_test.go

```diff
@@ -43,3 +43,71 @@ func TestStepExport(t *testing.T) {
 		t.Fatal("bad")
 	}
 }
+
+func TestStepExport_OutputPath(t *testing.T) {
+	type testCase struct {
+		Step     *StepExport
+		Expected string
+		Reason   string
+	}
+	tcs := []testCase{
+		{
+			Step: &StepExport{
+				Format:         "ova",
+				OutputDir:      "output-dir",
+				OutputFilename: "output-filename",
+			},
+			Expected: "output-dir/output-filename.ova",
+			Reason:   "output_filename should not be vmName if set.",
+		},
+		{
+			Step: &StepExport{
+				Format:         "ovf",
+				OutputDir:      "output-dir",
+				OutputFilename: "",
+			},
+			Expected: "output-dir/foo.ovf",
```

Comment:
I guess you need to use the `filepath.Join(...)` here to use the right OS path separator.